### PR TITLE
Mongo portfolio

### PR DESCRIPTION
**Pull request by natehood96**
This is the portfolio section of our database using MongoDB

### CODE REVIEW

File: server/src/db/mongo/portfolio.ts [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+      description, file, businessId
+    })
+    return result.insertedId.toHexString()
+  } catch (err){
+    console.error('Unexpected error inserting portfolio', err)
+    throw new Error('FailedInsertPortfolioItem')
+  }
+}
+
+interface Portfolio {
```
Comment (jasonccox) [inline server/src/db/mongo/portfolio.ts]: Can we call this `PortfolioItem` instead?

File: server/src/db/mongo/portfolio.ts [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  description: string
+  businessID: string
+}
+
+export async function get(businessId: string): Promise<Array<Portfolio>> {
+  let result : Portfolio[] = [];
+  try{
+    result = await getDB().collection("portfolio").find(
+      { businessId: businessId}
+    ).toArray();
```
Comment (jasonccox) [inline server/src/db/mongo/portfolio.ts]: I realized that we probably do want to have the id on here as a string just like we did in the PostgreSQL version. This should do the trick:
```suggestion
    ).toArray().map(item => ({
      id: item._id.toHexString(),
      file: item.file,
      description: item.description,
    }));
```

File: server/src/db/mongo/portfolio.ts [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    console.error('Unexpected error inserting portfolio', err)
+    throw new Error('FailedInsertPortfolioItem')
+  }
+}
+
+interface Portfolio {
+  id: string
+  file: string
+  description: string
+  businessID: string
```
Comment (jasonccox) [inline server/src/db/mongo/portfolio.ts]: I think we need a lowercase "d".
```suggestion
  businessId: string
```

File: server/src/db/mongo/portfolio.ts (hunk 2 above)
Comment (jasonccox) [reply server/src/db/mongo/portfolio.ts]: Then you can add the id back into your tests if you want.

File: server/src/db/mongo/portfolio.ts (hunk 3 above)
Comment (jasonccox) [reply server/src/db/mongo/portfolio.ts]: jk

File: server/src/db/mongo/portfolio.ts (hunk 1 above)
Comment (jasonccox) [reply server/src/db/mongo/portfolio.ts]: Actually, just use `Portfolio` from `src/types/portfolio.ts` instead

File: server/src/db/mongo/portfolio.ts (hunk 2 above)
Comment (natehood96) [reply server/src/db/mongo/portfolio.ts]: Done!

**Pull request merged**